go.mod

```diff
@@ -5,9 +5,13 @@ go 1.21
 toolchain go1.21.0
 
 require (
+	github.com/containerd/containerd v1.7.15
 	github.com/go-git/go-billy/v5 v5.5.0
 	github.com/go-git/go-git/v5 v5.12.0
 	github.com/go-logr/logr v1.4.1
+	github.com/google/go-containerregistry v0.19.0
+	github.com/google/go-containerregistry/pkg/authn/k8schain v0.0.0-20240418155129-98dd3e91704f
+	github.com/google/go-containerregistry/pkg/authn/kubernetes v0.0.0-20240418155129-98dd3e91704f
```

Comment:
These imports are confusing, `authn` package is already in `go-containerregistry` why are we pulling in 2 different packages explicitly? Or is this just happening automatically because those directories have mod files?

Either way, I think we'd want to go go `0.19.1`, since that is out and latest.

Reply:
These were pulled automatically via `go mod tidy`

Reply:
Also, authn/k8schain and authn/kubernetes have their own mod files and don't have tags. These commits align with the version of go-containerregistry in the go.mod (which for this PR is 0.19.0). I can bump it to 0.19.1 if necessary

Reply:
Yeah, I just noticed they have their own mod files, which seems a strange choice that I've never noticed working with that project.

Reply:
bumped go-containerregistry version to 0.19.1 in https://github.com/operator-framework/rukpak/pull/874/commits/414a3791d5d936f7943f5f62fee3637af7010f91